pyfar/plot/shortcuts/shortcuts.json

```diff
@@ -39,10 +46,10 @@
         "key": ["down"],
         "info": "y-axis view downwards"},
     "move_cm_up": {
-        "key": ["+"],
+        "key": ["+", "ctrl+shift+up"],
         "info": "move colormap range up"},
     "move_cm_down": {
-        "key": ["-"],
+        "key": ["-", "ctrl+shift+down"],
```

Comment:
The hotkeys `+, _` are not very intuitive for me, I would assume that `+, -` are used for zooming.

Reply:
see above :)